### ISSUE TITLE
Multiple screen

### ISSUE THREAD

**Issue by Simonv92** [psieg/Lightpack#280]
Hi,
Does this program works with multiple screen setup? 
Thank you! 
Simone

**Comment by psieg** [comment]
In general, yes. You'll have to provide more details for a more precise answer. Which OS, how many screens, which arrangement?

**Comment by BlackX777** [comment]
I have Windows 10 and two Monitors with two times Adalight + Arduino Nano
one for Gaming one for Movies with different LED Setups.
Proplem is the Profiles don't change the SerialPort (COM3 & COM4)
so I must everytime switch the main.conf .
I use Prismatik (unofficial) 5.11.2.19 with added NvFBC support :P
Running two different Instances at the same time also dont work respectively they use the same config.
Is it possible to implement support for more Adalights?

with best regards Toni

**Comment by gastonMM** [comment]
any solution to have two screens? I have my PC monitor and the TV

**Comment by BlackX777** [comment]
maybe you simply split the USB Signal/Kable

**Comment by akincansenol** [comment]
I also looking forward to using my multiple monitor setup with their adalight setup.  

I was able to run two instances via shift+right click "Run as different user" on Windows 10 2004 but somehow it's not working now. When I try to run second instances on a different user, main prismatik user detail in task manager changes to another user. So there aren't two instances at the same time with this method now.

**Comment by Attack44** [comment]
I don't use a multi monitor setup, but multiple instances for multiple devices / configs would be great.

**Comment by polykoma** [comment]
Hey buddy, great to see someone is here to improve Prismatik.

I also wanted to ask if you can enable Prismatik for multiple devices! I would donate you straight cash if you do so!

Im missing this feature for years. I use Ambibox on main screen and prismatik on second. But I got a 3 screen setup - and I hate one screen beeing dark.

Best Regards from Germany!

**Comment by polykoma** [comment]
> 
> 
> In general, yes. You'll have to provide more details for a more precise answer. Which OS, how many screens, which arrangement?

Hey buddy, great to see someone is here to improve Prismatik.

I also wanted to ask if you can enable Prismatik for multiple devices! I would donate you straight cash if you do so!

Im missing this feature for years. I use Ambibox on main screen and prismatik on second. But I got a 3 screen setup - and I hate one screen beeing dark.

(Config: LPD8806 Stripes behind all Screens with 88 LED Each / 3x Arduino nano / Ambibox on Main screen / Prismatik on Second / Third screen remains dark because unsupported yet)

Best Regards from Germany!

**Comment by psieg** [comment]
It's not obvious to me whether it would be better to be able to run multiple instances at the same time with different devices or be able to merge multiple devices into one. This is implemented for the original Lightpack hardware.

Multiple instances sounds inefficient, the advantage would be running different things (e.g. capture one one vs moodlight on the other). Is that something anyone here would want to do (rather than treating their 3 devices as one?)

**Comment by BlackX777** [comment]
Mh maybe Multiple instances is easier to Develop. E.g you have different Monitors with other resolutions one 4:3 and so on. Also on modern Hardware where you have multiple Cores that won't be a problem?

**Comment by polykoma** [comment]
> It's not obvious to me whether it would be better to be able to run multiple instances at the same time with different devices or be able to merge multiple devices into one. This is implemented for the original Lightpack hardware.
> 
> Multiple instances sounds inefficient, the advantage would be running different things (e.g. capture one one vs moodlight on the other). Is that something anyone here would want to do (rather than treating their 3 devices as one?)

Hey there, I'm a pretty long term Ambilight user meanwhile and from my POV it would be a lot simpler to just modify it to be able to run several instances with several unique config files. Actually till a certain windows update you could bypass it and were already able to run 2 instances of it. So its capable of it just needs some minor tweaks. Also in regards of usability and bugfixing sw/hw it would be great to have it separated.  Thank you for looking into the issue ! Im dreaming about multi monitor capability since I build my first ambilight :)

**Comment by zomfg** [comment]
try re-running wizard on todays release to see if it fits some of your needs

**Comment by polykoma** [comment]
> try re-running wizard on todays release to see if it fits some of your needs

Hey there :) Is your answer regarded to multi monitor setup? Then i have to check the release asap 👍

**Comment by zomfg** [comment]
it is

**Comment by polykoma** [comment]
Are you able to drive 3 different Arduinos COM3 COM5 an COM7 and have an individual setup with only one instance of prismatik?
I still cant believe this works now. Im downloading it right now

**Comment by zomfg** [comment]
not exactly, this is still single device AND single mode (grab/lamp/sound), but you can setup individual displays
so if you were using 3 arduinos to capture 3 displays, this should work with only 1 (but rewiring is probably required)

technically this always worked, there was just no easy way to setup all displays

**Comment by polykoma** [comment]
Thats why I mention to just let us run several instances of prismatik. Each instance has its own config wizard and you setup each arduino behind each screen directly. That would be a blast. If you have a satelitte screen for example you run content on with separated ambilight.
I got 3 screen with 3 full plastered leds each individually wired to an arduino behind it. (1x prismatik 1x ambibox 1x nothing)
Anyway I will probably have to rewire at least one setup and pull one arduino out from the not yet ambilight screen.

I just run through the new setup wizard and I cant finish it
![grafik](https://user-images.githubusercontent.com/11220003/101294044-1a33c080-3815-11eb-9bae-cf9aa2e325fe.png)

COM port ist correct. I did this setup 50 times in the past and somehow this occurs now

**Comment by zomfg** [comment]
it's a bug... see you in the next build...
leaves you some time for wiring!

**Comment by polykoma** [comment]
Thank you very much kind Sir for answers and time! Gonna rewire meanwhile! Really happy to see (ambi)light at the end of the tunnel.(on all of my screens)

**Comment by psieg** [comment]
Oh. Sorry, multiscreen layouts (with a single device) have been working from the beginning. Sorry that that wasn't clear -.-

**Comment by polykoma** [comment]
> 
> 
> Oh. Sorry, multiscreen layouts (with a single device) have been working from the beginning. Sorry that that wasn't clear -.-

So I basically just continue the led string with extension cables to the next monitors string?
Before I rewire and maybe misundestood something. Can you please shortly confirm if below drawing would be working?
Thank you so much!
![Unbe12312nannt](https://user-images.githubusercontent.com/11220003/101398837-faf96980-38ce-11eb-8859-e63545926920.JPG)

**Comment by psieg** [comment]
I don't have such a setup running to be 100% sure, but yes it *should*. If your device is powerful enough for the 264 LEDs and bandwidth isn't an issue (I don't have experience with Adalight at all).

To be sure about the Software side, you could set up a virtual device and arrange the zones as desired and make sure the everything works.

**Comment by polykoma** [comment]
I need to wait till next build, current one has a bug in the setup manager as you saw above.
I also need some time to put extra wires on to confirm if it would be able to work.

Also I have to mention I would only need 2 screen driven by prismatik (1 + 3) so 176 total which should be fine bandwidth size.
Screen 2 stays separate and will be driven by ambibox on a separate arduino.

**Comment by psieg** [comment]
yes, try this: https://github.com/psieg/Lightpack/releases/tag/5.11.2.27

**Comment by polykoma** [comment]
Awesom! Setup manager works again! Thank you. Once I got my wiring correct I will give you feedback!
Thanks a ton for figuring this out with me - basically an early christmas for me :) <3

**Comment by burnbabyburn** [comment]
thx for keeping prismatik alive!
I am currently rebuilding my leds and i wanted to use multi mon setup too.
IMHO: daisy chaining multiple monitors led strips is the worst possible solution. Some may get stuck with bandwith, other setups fail because they just rip off the cables while moving monitors. (dual monitor stand anyone?)
Added cable length on the full path also won't make ws2812b happy.
Wiring would be messy with a flying connection between monitors or more cable length
so it would be nice if we could use multiple devices/instances in the future.

I've also linked a issue where someone is aksing for multiple udp output devices.

@polykoma 
_hat es denn bei dir, so wie in der Zeichnung, geklappt stefan?_ / Did your drawing above work for you polykoma?

**Comment by polykoma** [comment]
I dont like the idea of daisy chaining and would love a standalone (1x arduino per screen and string) better aswell.

Sorry ich kann noch nicht bestätigen ob es wirklich klappt. Ich nutze LPD88806 LEDS.  Ich sollte in den nächsten 2 wochen die leds für den dritten screen bekommen und kann dann sagen ob es wirklich geht. Das setup sah zumindest so aus als würde es gehen :)

**Comment by burnbabyburn** [comment]
@polykoma 
hat funktioniert. Man richtet den beide monitore normal ein und muss den offset für den zweiten monitor eintragen/ The above drawing from polykoma works fine. each display can be configured with its own led settings. It just asks for the offset (# of leds on previous monitor) an worked!

**Comment by polykoma** [comment]
@burnbabyburn Cool danke fürs bestätigen! Ich nutze am Hauptmonitor Ambibox das finde ich besser. Aber meine beiden Seitenscreen sollen dann auf prismatik laufen. Blos fehlen mir gerade noch die Leds für den screen3.

**Comment by polykoma** [comment]
> @polykoma
> hat funktioniert. Man richtet den beide monitore normal ein und muss den offset für den zweiten monitor eintragen/ The above drawing from polykoma works fine. each display can be configured with its own led settings. It just asks for the offset (# of leds on previous monitor) an worked!

Wie hast du die strings miteinander verbunden ? Ich bin gerade fertig mit löten und habe ein geschirmtes  netzwerkkabel ca 2,5 meter lang zwischen den monitoren um Data Out und Clock Out mit dem nächsten string zu verbinden.
Sobald ich strom auf den frisch montierten stripe gebe leuchten alle weiß und flackern ab und zu. Ich habe dann das setup in prismatik ausgeführt aber er scheint den neuen stripe nicht mit anzusteuern.
Den startpunkt habe ich richtig gesetzt. Setup was ja recht eindeutig in prismatik.
Ich versuche morgen mal ein kürzeres kabel.

**Comment by polykoma** [comment]
Do I have to daisy chain the ground cable as well or only Data and Clock?
I managed to setup 2 displays in prismatik and connected them via a 3m cat6 cable (data and Clock) but some of the LED keep flashing in random color.
Both strings drive by an individual power supply. Do i have to daisy chain the ground wire aswell?

**Comment by sblantipodi** [comment]
daisy chaining is not the best solution imho, multiple instances is far better. wiring on multiple monitors is a mess.

**Comment by polykoma** [comment]
> 
> 
> daisy chaining is not the best solution imho, multiple instances is far better. wiring on multiple monitors is a mess.

I just wired up the ground from the first to second string and now it works... Yeah its annoying but its just one more cable and thats it. Now I have to redo a couple of custom cables i already put together. Ahh what a pain...

**Comment by burnbabyburn** [comment]
Wie ich sehe hast du die Lösung bereits selbst gefunden. Hatte bereits den ganzen Text für Pufferkondensator,Widerstand und geteilter Masse geschrieben bis ich gesehen habe, dass du es bereits selbst gelöst hast :)

**Comment by Janoo1606** [comment]
Hi,
i have also two monitors with different devices on different COM ports.
Would it simply be possible to create a Prismatik installation with a different name and different config folder?
This would be the best solution for me and maybe some others.
Unfortunately, I don't know how time-consuming such a change would be.
But maybe it's just a small change of the program name and the folder names.

**Comment by psieg** [comment]
I finally had some time to look at this. The multiple-instances thing is rather simple. There's an experimental build [here](https://github.com/psieg/Lightpack/suites/2749786233/artifacts/60804647), please give it a try. You can run the second instance with `Prismatik.exe --config-dir C:\Users\YourName\Prismatik2` for example. Also happy for reviews of the [Pull Request](https://github.com/psieg/Lightpack/pull/462/files).

Note: I would still recommend to use one device with the strip covering all screens though.

**Comment by Janoo1606** [comment]
Thanks you for your effort! :)
I will try it.

**Comment by Janoo1606** [comment]
It works in principle, but unfortunately one of the two programs terminates after a short time. (The icon in the system tray disappears). I also tried it with administrator rights. I suspect that Windows only allows one instance of the same program. I am using Windows 10.

**Comment by psieg** [comment]
Ah, I found a crash from an unintended interaction between multiple instances. Give [this](https://psieg.de/jenkins/job/Prismatik_Windows/685/artifact/Software/dist_windows/Output/Prismatik%20(unofficial)%2064bit%20Setup%205.11.2.27.exe)  a try please.

**Comment by Janoo1606** [comment]
The first attempt looks good so far.
Thank you so much!

**Comment by TylersSavage** [comment]
Lmao, I just spent several weeks￼[upgrading my living room](https://streamable.com/zyrzqk).. I'm just about done, then I see this post￼. 

Patrick, is there any way to contact you? I want to make sure you get credit, and I don't mislead anyone. My first setup has been seen around 200k times now, because Dustin Farrell (who made the Transient lightning videos) shared it. He's going to share this one too. 

Sorry if this post doesn't belong here, I have been trying to find a way to contact you. I'll remove it if necessary. 

Thank you!
@tylerssavage (IG)

**Comment by psieg** [comment]
Hi Tyler, glad Prismatik is useful for you! You can email me at github@psieg.de. In terms of credit, just point people to the repo here ;)

On-topic, I made an official release with the multi-instance mode: https://github.com/psieg/Lightpack/releases/tag/5.11.2.28

**Comment by TylersSavage** [comment]
Thank you very much!

**Comment by freakinside** [comment]
> Hi Tyler, glad Prismatik is useful for you! You can email me at [github@psieg.de](mailto:github@psieg.de). In terms of credit, just point people to the repo here ;)
> 
> On-topic, I made an official release with the multi-instance mode: https://github.com/psieg/Lightpack/releases/tag/5.11.2.28

Is this included in v6 ? It says "Unknown option config-dir"

**Comment by psieg** [comment]
There is no v6 from this fork. There is an official v6 version of which I don't know what it contains.

**Comment by freakinside** [comment]
I noticed that after researching a bit more. Thanks a lot for your version. It's awesome (i really needed multiple instances).

Btw i missed the option to add different margins to left and right as i just use a part of the screen that is not centered (4 zones in screen = 4 instances). I'll edit the positions manually on the profile file.

**Comment by jafet420** [comment]
i just got another monitor installed and was wondering how u guys are wiring up the second monitor?

**Comment by burnbabyburn** [comment]
> i just got another monitor installed and was wondering how u guys are wiring up the second monitor?

[See daisy chain solution](https://github.com/psieg/Lightpack/issues/280#issuecomment-740146304)
[See second instance solution](https://github.com/psieg/Lightpack/issues/280#issuecomment-841696633)

For the daisy chain solution there is a pciture attached to the post. For a second instance, well you take two arduinos and connect one strip each. Than run an instance per arduino

**Comment by GreenDie** [comment]
I tried manyx hours to connect two ESP32 devices with WLED . I have one for tzhe LEDs behind my TV and one in my room.
I cant connect them on one ESP.

Where do i have to enter this Code?
`--config-dir C:\Users\Daniel\Prismatik3`

I have a folder for the LED's behind the TV and one for them behind my couch.
But how do i tell prismatic to run both folders?

**Comment by jordanadania** [comment]
This doesn't work for me.  The first config is used for both instances. When I go to run Wizard AGAIN because it's using the wrong COM port (setup took an hour [im mapping 256 leds and each one needs a custom location]) it only shows "Profile to modify:" profiles from the first instance. I checked the folders and files manually and they are correct.  If anyone knows how to fix this, please let me know.

**Comment by KarloLeksic** [comment]
> Lmao, I just spent several weeks ￼[upgrading my living room](https://streamable.com/zyrzqk).. I'm just about done, then I see this post￼. 😂
> 
> Patrick, is there any way to contact you? I want to make sure you get credit, and I don't mislead anyone. My first setup has been seen around 200k times now, because Dustin Farrell (who made the Transient lightning videos) shared it. He's going to share this one too.
> 
> Sorry if this post doesn't belong here, I have been trying to find a way to contact you. I'll remove it if necessary.
> 
> I appreciate all the hard work, and I'm loving it!
> 
> Thank you ⚡️ @TylersSavage (IG)

Are there any detailed instructions to make a setup like this😍? I made ambilight on my monitor using Adalight on Arduino Nano and Prismatic. How to add other devices via WiFi that are above or beside the monitor. I am familiar with WLED on ESP, but I don't know how to add it in Prismatic and synchronize it with the monitor.

**Comment by v301eu** [comment]
Good morning,
Has anyone managed to run these two instances of Prismatik yet? no. Com1 and Com3?

USB2.0 Device 1,   Prismatik   , LED Stripe RGB (red green blue)     5V 3Pin direct to Led stripe
USB2.0 Device 2,   Prismatik2 , LED Stripe RBG (red blue green)     5V 3Pin connected to 5V TTL signal amplifier up 12Volt

The CMD Administrator command stil does not work   sadge(
"using the --config-dir command line option"    sadge(

C:\Users\webud>Prismatik.exe --config-dir C:\Users\webud\Prismatik2
'Prismatik.exe' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\webud>cd: C:\--5309547

 Directory of C:\Program Files\Prismatik

31/03/2023  15.16    <DIR>          .
31/03/2023  09.41    <DIR>          ..
08/01/2022  13.17           257.536 bass.dll
08/01/2022  13.17            27.136 basswasapi.dll
08/01/2022  13.09            35.821 gpl-3.0.txt
08/01/2022  13.17         3.394.560 libcrypto-1_1-x64.dll
08/01/2022  13.17            17.408 libraryinjector.dll
08/01/2022  13.17           679.424 libssl-1_1-x64.dll
08/01/2022  13.17           565.648 msvcp140.dll
08/01/2022  13.17            23.952 msvcp140_1.dll
08/01/2022  13.17           185.720 msvcp140_2.dll
08/01/2022  13.17            56.208 msvcp140_atomic_wait.dll
08/01/2022  13.17            20.368 msvcp140_codecvt_ids.dll
08/01/2022  13.17            85.504 offsetfinder.exe
28/03/2023  04.27    <DIR>          platforms
08/01/2022  13.17           140.800 prismatik-hooks.dll
08/01/2022  13.17           103.424 prismatik-hooks32.dll
08/01/2022  13.17           104.448 prismatik-unhook.dll
08/01/2022  13.17            80.896 prismatik-unhook32.dll
08/01/2022  13.17         2.051.584 Prismatik.exe
08/01/2022  13.09           370.070 Prismatik.ico
08/01/2022  13.17         6.023.664 Qt5Core.dll
08/01/2022  13.17         7.008.240 Qt5Gui.dll
08/01/2022  13.17         1.340.400 Qt5Network.dll
08/01/2022  13.17            75.760 Qt5SerialPort.dll
08/01/2022  13.17         5.498.352 Qt5Widgets.dll
28/03/2023  04.27    <DIR>          styles
28/03/2023  04.27             7.649 unins000.dat
28/03/2023  04.27         3.501.629 unins000.exe
08/01/2022  13.17            50.176 UpdateElevate.exe
08/01/2022  13.17            97.168 vcruntime140.dll
08/01/2022  13.17            37.240 vcruntime140_1.dll
              28 File(s)     31.840.785 bytes
               14 Dir(s)  789.382.584.815.616 bytes free

**Comment by TheMcSebi** [comment]
Worth noting that for me to be able to grab the screen of two different monitors I needed to run one of the instances as administrator.

**Comment by raflyramandani26** [comment]
> Good morning, Has anyone managed to run these two instances of Prismatik yet?
> 
> COM3 USB2.0 Device 1, Prismatik , LED Stripe RGB (red green blue) 5V 3Pin direct to Led stripe COM4 USB2.0 Device 2, Prismatik2 , LED Stripe RBG (red blue green) 5V 3Pin connected to 5V TTL signal amplifier up 12Volt
> 
> Conclusion: "I think the creator code cpp needs to change the program code so that after a full cycle counts all pixels from the screen, change the colour of sequence = type2 RBG, until the pixels from the monitor are back again cycle RGB. In the Loop."
> 
> Windows PowerShell Copyright (C) Microsoft Corporation. All rights reserved.
> 
> Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows Add path system environment variables before use PowerShell or CMD: `C:\Program Files\Prismatik\` (Regardless of whether User or Administrator).
> 
> ```
> PS C:\Users\UserName> prismatik.exe --config-dir C:\Users\YourUserName\Prismatik2
> PS C:\Users\UserName> 
> ```
> 
> ```
> PS C:\Users\UserName> prismatik.exe --config-dir C:\Users\YourUserName\Prismatik3
> PS C:\Users\UserName> 
> ```
> 
> ![Prismatik multi instances RGB RBG](https://user-images.githubusercontent.com/116048607/229186916-51735a96-9fc4-42c8-91db-d4af32a2a963.jpg)
> 
> Requirer SAVE two different Profiles name, otherwise it will crash and you will have to delete the Prismatik and Prismatik2 folders from User, and do the configurations again.
> 
> ![Two instances](https://user-images.githubusercontent.com/116048607/229201682-8afc56d2-61d3-4903-a143-7621a6bf6510.jpg)
> 
> Remember to add 60mAh for the 5050 SMD diode during configuration, and if you use the monitor's USB power supply, the BC1.2 socket, enter the maximum power consumption of 2,4 amperes 5V = 12Watt so as not to crash the LED controller.
> 
> Autostart: Prismatik 1 (default) + 2, 3, ... ![startup](https://user-images.githubusercontent.com/116048607/231029366-483fc622-3717-44ab-8ff5-c74f1b66bb01.jpg)

hello, i've followed your tutorial and it's work, but if i turn on the computer, the wizard prismatik2 always pop up, and then i must setup the led again, profile which i have it's not saved, how to resolve that? maybe i missed something? thank you

**Comment by v301eu** [comment]
"hello, i've followed your tutorial and it's work, but if i turn on the computer, the wizard prismatik2 always pop up, and then i must setup the led again, profile which i have it's not saved, how to resolve that? maybe i missed something? thank you"

_You must use "your username" in the path, during you create the Start-up shortcut for Prismatik2_

![shortcut](https://user-images.githubusercontent.com/116048607/236611035-0678a07e-862b-4c35-8d12-e9679d4ba1c0.jpg)

`prismatik.exe --config-dir C:\Users\YourUserName\Prismatik2`

**Comment by raflyramandani26** [comment]
> "hello, i've followed your tutorial and it's work, but if i turn on the computer, the wizard prismatik2 always pop up, and then i must setup the led again, profile which i have it's not saved, how to resolve that? maybe i missed something? thank you"
> 
> _You must use "your username" in the path, during you create the Start-up shortcut for Prismatik2_
> 
> ![shortcut](https://user-images.githubusercontent.com/116048607/236611035-0678a07e-862b-4c35-8d12-e9679d4ba1c0.jpg)
> 
> `prismatik.exe --config-dir C:\Users\YourUserName\Prismatik2`

yes i did it, but the profile can't be saved and the folder of prismatik2 not automatic create

**Comment by v301eu** [comment]
_"yes i did it, but the profile can't be saved and the folder of prismatik2 not automatic create"_

`My ver. software 5.11.2.31 (rev [5044e21](https://github.com/psieg/Lightpack/commit/5044e21), Qt 5.15.2)`

I use this version for me it works in win11, if you find a solution to the problem, please add a post.

**Comment by raflyramandani26** [comment]
> _"yes i did it, but the profile can't be saved and the folder of prismatik2 not automatic create"_
> 
> `My ver. software 5.11.2.31 (rev [5044e21](https://github.com/psieg/Lightpack/commit/5044e21), Qt 5.15.2)`
> 
> I use this version for me it works in win11, if you find a solution to the problem, please add a post.

same as you, i'm using win11 and latest version of prismatik, oh ya my config using warls UDP, could this be a problem?

**Comment by polykoma** [comment]
Yo guys. Funnily I switched from 2 monitor daisy chain and third monitor with own arduino running ambibox to  double daisy chain.

So basically like in my drawing here above in my post.
https://github.com/psieg/Lightpack/issues/280#issuecomment-740146304

But I can NOT get anything to work after screen 2. I got multiple LPD8806 stripes here and tried 3 new ones + a short piece of my old one. and i just can not get anything to work after my second piece of stripe in the daisy chain. everything has contact and i tried switchting data clock etc everything. also measured all contact with multimeter, no cold spots. ran the wizard on 2 differen prismatik versions multiple times. i power it and its just full white on max and doesnt respond to prismatik or ambibox.
its the same chipset but different manufacturing years. but on the other hand i cant get my 2 leftover pixels from exactly that already working 2 monitors. doesnt light up aswell

also tried connecting the strip without any extending daisy chain cable directly on the end but there is just no data exchange

Sidenote: when i power my new lpd8806 stripe without any other connections. most of the time it just start lighting full white on max. while my old leftover piece doesnt light up at all.

if there is any ultra PRO here reading this. Willing to earn a dime helping me getting this figured I would be extremely gratefull

**Comment by polykoma** [comment]
UpdATE: 
Long story short.

I literally have 2 types of LPD8806 / 60Leds meter stripes and they are not compatible with each other. Even they look the same. Its hilarious. What a waste of my time. 1 Week down the drain lol.

Upgrading everything to SK6812 RGBW now since i found working code for those.

**Comment by v301eu** [comment]
> UpdATE: Long story short.
> 
> I literally have 2 types of LPD8806 / 60Leds meter stripes and they are not compatible with each other. Even they look the same. Its hilarious. What a waste of my time. 1 Week down the drain lol.
> 
> Upgrading everything to SK6812 RGBW now since i found working code for those.

The simplest way is to use two USB 1 and USB 2 devices, helpful to invert the RGB BGR in two different addressable chips.

**Comment by polykoma** [comment]
> > UpdATE: Long story short.
> > I literally have 2 types of LPD8806 / 60Leds meter stripes and they are not compatible with each other. Even they look the same. Its hilarious. What a waste of my time. 1 Week down the drain lol.
> > Upgrading everything to SK6812 RGBW now since i found working code for those.
> 
> The simplest way is to use two USB 1 and USB 2 devices, helpful to invert the RGB BGR in two different addressable chips.

Uhhh, sorry but I didnt quite understood that.

Do you mean running 1 Arduino for each Chipset?
I ended up using Ambibox on one screen with 1 Arduino for the one lpd chipset
 and then prismatik on the other 2 screen utilizing another arduino uno

But as said. In the process of changing all 3 screen to sk6812 rgbw. 1 is already done.
So 3x 110 Leds each screen soon. :)

**Comment by M0NeTKa** [comment]
> > "hello, i've followed your tutorial and it's work, but if i turn on the computer, the wizard prismatik2 always pop up, and then i must setup the led again, profile which i have it's not saved, how to resolve that? maybe i missed something? thank you"
> > _You must use "your username" in the path, during you create the Start-up shortcut for Prismatik2_
> > ![shortcut](https://user-images.githubusercontent.com/116048607/236611035-0678a07e-862b-4c35-8d12-e9679d4ba1c0.jpg)
> > `prismatik.exe --config-dir C:\Users\YourUserName\Prismatik2`
> 
> yes i did it, but the profile can't be saved and the folder of prismatik2 not automatic create

someone find the solution?

**Comment by raybantracing** [comment]
I was a big fan of the project but I must admit that prismatik seems abandoned currently.

I found a fix and various improvements with the Luciferin project.

https://github.com/sblantipodi/firefly_luciferin/wiki/Multi-monitor-support

https://github.com/sblantipodi/firefly_luciferin/wiki/Surround-lighting-with-satellites

**Comment by jordanadania** [comment]
The readme sends us here, but the explanation is not good. I have created a folder "prismatik2" for separate COM config. I have created a shortcut `"C:\Program Files\Prismatik\Prismatik.exe" --config-dir C:\Users\Jordan\Prismatik2` I have opened 2 instances. But running configuration wizard on 2nd instance overwrites the first instance. Running configuration wizard with 2 instances open causes both to crash. I would love more information.

**Comment by TheMcSebi** [comment]
Yeah, I noticed that, too. My approach was to finish configuration on one monitor, then rename the folder and continue with the next one by only using the config wizard on the main instance. Not great, but works.

Nowadays I've switched to Hyperion.

**Comment by v301eu** [comment]
Alternatively, can run configuration wizard gui and change USB name manually.
Run on second wizard and clode window on second monitor is remember with one it is.
________________________________

**Comment by jordanadania** [comment]
@v301eu I can't find how to manually change the COM port, is that what you mean?

@TheMcSebi That sounds like it's still worth doing in my case. Could I just move the configuration from the first wizard to the second folder? Instead of renaming the folder? I guess I can just try it instead of asking. Thanks for the tip.

**Comment by v301eu** [comment]
In creator wizard GUI open twice once window 1 monitor, 2 second monito and create shortcut.
https://www.aliexpress.com/i/1005001728062595.html

[https://ae01.alicdn.com/kf/U75ba47cb45794e88b7795d437e3884ceu.jpg]<https://www.aliexpress.com/i/1005001728062595.html>
Dream Color RGB Ambient TV PC Screen Back Led Lighting WS2812b USB Led Strips Tape For Android 5.1+ System TV Box/ PC Backlight - AliExpress 39<https://www.aliexpress.com/i/1005001728062595.html>
Smarter Shopping, Better Living! Aliexpress.com
www.aliexpress.com

[https://ae01.alicdn.com/kf/U75ba47cb45794e88b7795d437e3884ceu.jpg]<https://www.aliexpress.com/i/1005001728062595.html>
Dream Color RGB Ambient TV PC Screen Back Led Lighting WS2812b USB Led Strips Tape For Android 5.1+ System TV Box/ PC Backlight - AliExpress 39<https://www.aliexpress.com/i/1005001728062595.html>
Smarter Shopping, Better Living! Aliexpress.com
www.aliexpress.com

________________________________
